### DISCUSSION TITLE
How to use loki async callback function with CSF in Angular? #332

### DISCUSSION THREAD

**Discussion by alexanderfink** [oblador/loki#334] [Q&A]
Hello, I moving my Issue #332 from the Issues tab to the discussion tab. I am still wondering how to implement an async callback in CSF Format or using the old storiesOf Format in **Angular.** 

I could not bring up a solution with @techeverri React examples from the repo. 

[https://github.com/oblador/loki/blob/8beda16fcda4841c4e23c7353880e94498cbadcc/examples/react-native/storybook/stories/index.js#L58-L60](https://github.com/oblador/loki/blob/8beda16fcda4841c4e23c7353880e94498cbadcc/examples/react-native/storybook/stories/index.js#L58-L60)

**Comment by d3vl1m3** [comment]
Very old thread now but I've found this solution works with React stories and it may apply to a suitable solution of other frameworks.

https://github.com/oblador/loki/discussions/384?sort=new
